tests/test_startup.py

```diff
@@ -245,26 +245,39 @@ def _mock_folder_listing(self):
         else:
             yield
 
-    def _get_plugin_environment(self, dcc_path):
+    def _join_existing_environment(self, dcc_path, startup_path):
+        """
+        Returns the pre-existing environment paths for dcc_path, os.pathsep joined with the startup_paths
```

Comment:
Instead of modifying the test so that it can handle when the environment variable is set prior to running the test maybe you could clear the environment variable from os.environ during setUp. Simply override [setUp](https://github.com/shotgunsoftware/tk-core/blob/master/tests/python/tank_test/tank_test_base.py#L193) from the base class (but don't forget to call it!)